setup.production.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Asking is better then forgetting
+function ask_if_docker_compose_file_has_been_updated() {
+    echo "Are the settings in the compose-production.yml correct and is compose-production.yml available through docker-compose.yml (symlink it)?"
```

Comment:
I guess it could, but then it would not be possible anymore to copy it, which sometimes might be what the user would want.